packages/elements/src/components/ino-list/ino-list.scss

```diff
@@ -1,11 +1,37 @@
 @use 'base/mdc-customize';
 @use '@material/list';
+
 @include list.core-styles;
 
+/**
+  * @prop --border-radius: Border radius of the list container
+ */
 ino-list {
+  --border-radius: (var(--border-radius), 0);
```

Comment:
Shouldn't it conform to the naming convention used in other components, i.e., `--list-item-border-radius: (var(--ino-list-item-border-radius), 0);`?

Reply:
Yes, you are right! I changed it. Could you review it again?